component/main.jsonnet

```
@@ -5,6 +5,49 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.metallb;
 
+local namespace = kube.Namespace(params.namespace) {
+  metadata+: {
+    labels: {
+      app: 'metallb',
```

According to our [labels guide](https://syn.tools/syn/references/labels-guide.html) I'd suggest the following:
```suggestion
      app.kubernetes.io/name: 'metallb',
      app.kubernetes.io/instance: 'metallb',
```

---

This is the default label metallb provides https://raw.githubusercontent.com/metallb/metallb/v0.9.3/manifests/namespace.yaml. Fine just to add ours?

---

Yes, usually it doesn't matter